backend/src/main/kotlin/com/infowings/catalog/data/objekt/ObjectService.kt

```diff
@@ -258,14 +261,28 @@ class ObjectService(
 
     private fun ObjectPropertyValueVertex.toValueResult() = ValueResult(
         this,
-        this.toObjectPropertyValue().value.toObjectValueData().toDTO(),
-        this.measure?.id,
+        this.toObjectPropertyValue().calculateObjectValueData().toDTO(),
+        this.explicitMeasure(),
         this.objectProperty ?: throw IllegalStateException("Object value was created without reference to object property"),
         this.aspectProperty,
         this.parentValue
     )
 
-    private fun ValueResult.toResponse() = ValueChangeResponse(id, valueDto, description, measureId, Reference(objectPropertyId, objectPropertyVersion),
+    private fun ObjectPropertyValueVertex.explicitMeasure(): String? {
+        val currentMeasure = this.measure?.toMeasure()?.name
+        return if (currentMeasure == null) {
+            val aspectProperty = this.aspectProperty
+            when (aspectProperty) {
+                null -> this.objectProperty?.aspect?.measureName
+                else -> aspectProperty.associatedAspect.measureName
+            }
```

Comment:
val aspect = this.aspectProperty?.associatedAspect ?: this.objectProperty?.aspect 
aspect?.measureName